### ISSUE TITLE
Dev Cam Menu either broken or incredibly difficult to get ahold of

### ISSUE THREAD

**Issue by FireKeep-HLA** [Facepunch/sbox-issues#2278]
### Describe the bug

In an era long before the one we currently live in, the use of dev cam would also allow you access to a special menu where you could blur stuff and it looked kind of cool. Today, it doesn't seem to work anymore. Cinematography has never been the same since.

The "TAB" key is supposed to bring up the menu, however trying it today it doesn't seem to do anything. I've attempted to load it up in a regular server, a private server, local game AND map addons, and just loading a map from console. No menu on any situation. My uneducated guess is that as development continued, the menu got left behind and is somehow now incompatible with the current version of s&box.

[It's a shame because with SFM not a major concern as of current development, devcam is *THE* de-facto tool for creating promotional material for your s&box content. (In my case, I'd like to make short films using it)]

### To Reproduce

1. Open a map on almost any kind of server setup imaginable
2. Press 'F2' to use the dev camera
3. Press 'TAB' to attempt to open the dev cam menu
4. Recoil in horror as the menu fails to appear on your monitor

### Expected behavior

As you press 'TAB,' the menu *should* appear, allowing you to change things like aperture, focal depth, or whatever the menu is supposed to allow you to change (it's been so long I've kind of forgotten)

### Additional context

I do say I've tried basically every addon/server setup, but I *may* have missed something. I should say that I don't think even if I missed something and there is somewhere it works, it shouldn't be this hard to access in the first place. (Also please make it work in multiplayer settings 😛)

**Comment by cr4yz** [comment]
I'm unable to repro: https://files.facepunch.com/crayz/1b0511b1/sbox_joYnTkzquA.mp4

Few things to try - the devmenu opens via the `+iv_score` bind, check to see if your tab is bound to that.  You have to hold down tab, don't just press it.  If still nothing, try verifying files?

**Comment by KayaDLX** [comment]
I've noticed this happening on some gamemodes where hold inputs are broken in some occasions. I have never been able to reproduce this issue, it just happens from time to time.

**Comment by FireKeep-HLA** [comment]
I can confirm TAB is bounded to +iv_score, and I have held down the TAB key for multiple seconds. I've verified the files and still no menu.

The fact you're able to use it just fine makes me a bit suspicious - is it somehow a Facepunch only thing? I'll see if I can get a few other people to open the menu on their end but I know for a fact other people have been having issues getting the menu to open as well.

**Comment by handsomematt** [comment]
> I've noticed this happening on some gamemodes where hold inputs are broken in some occasions. I have never been able to reproduce this issue, it just happens from time to time.

It's probably happening when you have low fps (specifically when fps is lower than tick rate), I've got an internal issue to fix that.

**Comment by KayaDLX** [comment]
Yep I'm pretty sure it was related to low framerate because it was only happening on my crappy laptop.

**Comment by FireKeep-HLA** [comment]
I've made a breakthrough!

After reinstalling the game and launching grandprix (one of our game addons), the dev cam menu now works! With that knowledge, I think we can re-frame this issue as dev cam menus not working with map addons.

Ideally, you should be able to use this regardless of addon type. Hopefully this gives us somewhere to go?

**Comment by DevulTj** [comment]
I'm closing this in favour of an issue we have internally. Thank you for reporting this 😄
